ffmpeg/lpms_ffmpeg.h

```diff
@@ -36,10 +45,16 @@ typedef struct {
 typedef struct {
     int frames;
     int64_t pixels;
+
+  // Returns
+  int ret; // return code for this segment
```

Comment:
Looks like there is some mismatch in indentation of fields within this struct

Reply:
Yep, the this field was actually unused at this point. Removed.